Server/mods/deathmatch/logic/CRPCFunctions.cpp

```diff
@@ -178,6 +179,28 @@ void CRPCFunctions::PlayerWeapon(NetBitStreamInterface& bitStream)
             Arguments.PushNumber(m_pSourcePlayer->GetWeaponType(uiSlot));
 
             m_pSourcePlayer->CallEvent("onPlayerWeaponSwitch", Arguments);
+
+            SViewerMapType& nearList = m_pSourcePlayer->GetNearPlayerList();
+
+            if (!nearList.empty())
+            {
+                static std::vector<CPlayer*> playersInNearList;
+                playersInNearList.reserve(nearList.size());
+                playersInNearList.clear();
+
+                for (const auto& player : nearList)
+                    if (player.first->CanBitStream(eBitStreamVersion::OnPlayerWeaponSwitch_Remote))
+                        playersInNearList.push_back(player.first);
```

Comment:
Can you use brackets **{ }** for multi-line *blocks*?

Reply:
Yes, done.